test/testdata/lsp/completion/constr.rb

```diff
@@ -0,0 +1,15 @@
+# typed: true
+
+extend T::Sig
+
+xs = T.let([], T::Array[Integer])
+
+# This completion result is a bit non-sensical at the moment (it doesn't makes
+# sense to return any results there) but at least it doesn't crash Sorbet like
+# it used to. If you made a change that improves this test, feel free to delete
+# this comment.
```

Comment:
What ends up being nonsensical about the completion result? The assertion on line 15 is for `each`, which doesn't seem crazy for a tab completion on the result of a `map`.

Reply:
arguably that would make sense if you asked for completion results at the end of the `each` method name, not after the `end` keyword. IMO it's kind of like asking for completion results right after a `}`: we should probably show you other results or no results. you're right that it's subjective—i was just trying to let people know that i'm not attached to the current completion results.